app.py

```diff
@@ -43,12 +43,17 @@ def list_from(item):
 class ProductDBInterface:
     """Interface object so that the UI can easily obtain filtered products."""
     def parse_sql_tuple(self, x):
-        return (ScrapedProduct(name = x[2], source = x[5], price = x[3], product_type=x[9], photo = x[4], instock= x[6], new = x[7], price_check= x[8]))
+        return (ScrapedProduct(msrp = x[4], name = x[2], source = x[6], price = x[3], product_type=None, photo = x[5], instock= x[7], new = x[8], price_check= x[9]))
```

Comment:
why did product_type switch to None here? everything else I follow.

Reply:
Ah, that was switched while debugging a different issue, I will fix this.

Reply:
I don't think I can do this until product_type is added as an attribute in the database...I will look into this

Reply:
I went ahead and added product_type into the db...I'm not sure how the old code worked without product_type being an attribute in the db though since this is indexing off the sql tuple of products?